source/features/what-changed-since-previous-release.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import fetchDom from '../libs/fetch-dom';
+import {getOwnerAndRepo} from '../libs/utils';
+import {isTagsListPage, isReleasesListPage, isSingleTagPage} from '../libs/page-detect';
+
+async function init(): Promise<void | false> {
+	if (select.exists('.blankslate')) {
+		return false;
+	}
+
+	const {ownerName, repoName} = getOwnerAndRepo();
+
+	// To extract the tag "v16.8.6" from links like "/facebook/react/releases/tag/v16.8.6"
+	const tagRegExp = new RegExp(`/${ownerName}/${repoName}/releases/tag/(.*)`);
+
+	// To extract the commit hash from links like "/facebook/react/commit/92a1d8feac32d03ab5ea6ac13ae4941f6ae93b54"
+	const commitRegExp = new RegExp(`/${ownerName}/${repoName}/commit/([0-9a-f]{5,40})`);
```

Comment:
These links never point to a different repo, so you just need

```suggestion
	const tagRegExp = /releases\/tag\/(.*)$/;
	const commitRegExp = /commit\/([0-9a-f]{40})$/;
```